style.css

```diff
@@ -0,0 +1,171 @@
+@media (min-width: 250px) {
+  * {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    background: #1c1a19;
+    font-family: 'Crete Round', serif;
+  }
+
+  header {
+    width: 100%;
+    height: 10vh;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    background-color: #3c3a39;
+  }
+
+  .porfolio_container {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .nameplateicons {
+    margin-left: 65vw;
+    gap: 3px;
+    width: 5vw;
+    justify-content: space-between;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .dropdown {
+    background: transparent;
+    border: none;
+  }
+
+  .dropdown:hover {
+    cursor: pointer;
+  }
+
+  .text {
+    color: #fff;
+    text-decoration: none;
+    width: 30vw;
+    white-space: nowrap;
+    margin-left: 6%;
+  }
+
+  .header_navigation {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+  }
+
+  .item_navigation {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    height: 80%;
+    background: #3c3a39;
+    padding: auto auto;
+    margin: auto 0;
+  }
+
+  .primary {
+    height: 124px;
```

Comment:
:red_circle: Kindly make sure you remove this property as it makes your main header overlaps with the role

|Your Design|Figma Design|
|:-----------:|:----------------:|
|![design](https://user-images.githubusercontent.com/26736582/212477033-36ef2a41-624f-4875-b704-312d5f9bd43c.png)|<img src="https://raw.githubusercontent.com/Santosh-Konappanavar/PortFolio/d59de3ae39135ae0990cb87876b1d6c6eacf9b91/image/Website%20snapshot.JPG"/>|